pkg/kubernetes/kubernetes.go

```diff
@@ -95,7 +95,7 @@ func (k *Kubernetes) LaunchEphemeralContainer(ctx context.Context, pod *corev1.P
 	debugContainer := &corev1.EphemeralContainer{
 		EphemeralContainerCommon: corev1.EphemeralContainerCommon{
 			Name:    ephemeralName,
-			Image:   fmt.Sprintf("nethax-probe:%s", ProbeImageVersion),
+			Image:   fmt.Sprintf("%s:%s", nethaxProbeContainer, ProbeImageVersion),
```

Comment:
`nethaxProbeContainer` should instead be a full on image reference, that is image name and tag, otherwise whatever value we pass in `nethaxProbeContainer` for an image should have the same tag a we define in `ProbeImageVersion`.